### PR TITLE
Update sv.json

### PR DESCRIPTION
**Pull request by magol**

### CODE REVIEW

File: web/public/locales/sv.json [hunk 1]
```diff
@@ -51,8 +51,11 @@
     "noLiveData": "Aktuell data är för tillfället ej tillgänglig för denna region",
     "dataIsDelayed": "Data för denna region kan vara upp till %s timme/timmar fördröjd. Aktuell data kan inte visas.",
     "estimated": "uppskattad",
+    "aggregated": "aggregerad",
     "dataIsEstimated": "En del data för denna region är uppskattad. Läs mer om vår uppskattingsmetodik <a href=\"https://github.com/electricityMap/electricitymap-contrib/wiki/Estimation-methods\" target=\"_blank\">här</a>.",
+    "exchangesAreMissing": "Överföringar är beaktade i beräkningarna, men visas inte för närvarande. Läs mer om våra historiska sammanställningar <a href=\"https://github.com/electricitymap/electricitymap-contrib/wiki/Historical-aggregates\" target=\"_blank\">här</a>.",
```
Comment (VIKTORVAV99) [inline web/public/locales/sv.json:56]:
```suggestion
    "exchangesAreMissing": "Överföringar är inkluderade i beräkningarna, men visas för närvarande inte. Läs mer om våra historiska sammanställningar <a href=\"https://github.com/electricitymap/electricitymap-contrib/wiki/Historical-aggregates\" target=\"_blank\">här</a>.",
```

File: web/public/locales/sv.json [hunk 2]
```diff
@@ -146,6 +149,7 @@
     "hideWindLayer": "Dölj vind-lager",
     "showSolarLayer": "Visa sol-lager",
     "hideSolarLayer": "Dölj sol-lager",
+    "weatherDisabled": "Väderdata är ej tillgänglig för denna tid",
```
Comment (VIKTORVAV99) [inline web/public/locales/sv.json:152]:
```suggestion
    "weatherDisabled": "Väderdata är ej tillgänglig för denna period",
```

File: web/public/locales/sv.json [hunk 3]
```diff
@@ -231,7 +235,7 @@
...
     "otherSources-question": "Varför visar ni inte andra källor för utsläpp förutom elproduktion?",
-    "otherSources-answer": "Det skulle vara utanför syftet med detta projekt. Men, på electricityMap arbetar vi aktivt med att utveckla en <a href=\"#whoAreYou\" class=\"entry-link\">ny lösning för att kvantifiera klimatpåverkan av dina vardagliga beslut</a>.",
+    "otherSources-answer": "Det skulle vara utanför syftet med detta projekt. Men, på electricity Maps arbetar vi aktivt med att utveckla en <a href=\"#whoAreYou\" class=\"entry-link\">ny lösning för att kvantifiera klimatpåverkan av dina vardagliga beslut</a>.",
```
Comment (VIKTORVAV99) [inline web/public/locales/sv.json:238]:
```suggestion
    "otherSources-answer": "Det skulle vara utanför syftet med detta projekt. Men, på Electricity Maps arbetar vi aktivt med att utveckla en <a href=\"#whoAreYou\" class=\"entry-link\">ny lösning för att kvantifiera klimatpåverkan av dina vardagliga beslut</a>.",
```

File: web/public/locales/sv.json [hunk 4]
```diff
@@ -249,11 +253,11 @@
...
     "feedback-question": "Kan jag ge er någon återkoppling?",
     "feedback-answer": "Gärna! Fyll i <a href=\"https://forms.gle/VHaeHzXyGodFKZY18\" target=\"_blank\">vårt återkopplingsformulär</a> eller <a href=\"mailto:app@electricitymap.org\">skicka oss ett e-brev!</a>",
     "contribute-question": "Kan jag bidra till ert projekt?",
-    "contribute-answer": "Ja! The Electricity Map är ett öppet källkodsprojekt vilket är möjligt tack vara våra volontärer. Om du vill hjälpa till att utveckla kartan, antingen genom att lägga till datakällor för nya regioner, lägga till nya funktioner eller fixa buggar, är du välkommen att gå med i vår <a href=\"https://github.com/corradio/electricitymap/\" target=\"_blank\">github</a>.",
+    "contribute-answer": "Ja! Appen är ett öppet källkodsprojekt vilket är möjligt tack vara våra volontärer. Om du vill hjälpa till att utveckla kartan, antingen genom att lägga till datakällor för nya regioner, lägga till nya funktioner eller fixa buggar, är du välkommen att gå med i vår <a href=\"https://github.com/corradio/electricitymap/\" target=\"_blank\">github</a>.",
```
Comment (VIKTORVAV99) [inline web/public/locales/sv.json:260]:
```suggestion
    "contribute-answer": "Ja! Appen är ett öppet källkodsprojekt vilket är möjligt tack vara våra volontärer. Om du vill hjälpa till att utveckla kartan, antingen genom att lägga till datakällor för nya regioner, lägga till nya funktioner eller fixa buggar, är du välkommen att gå med i vår <a href=\"https://github.com/electricitymap/electricitymap-contrib/\" target=\"_blank\">github</a>.",
```

Comment (magol) [reply web/public/locales/sv.json:260]: It is "https://github.com/corradio/electricitymap" in en.json. Should I change in the English translation too?

Comment (VIKTORVAV99) [reply web/public/locales/sv.json:260]: "https://github.com/corradio/electricitymap" redirects to "https://github.com/electricitymap/electricitymap-contrib" so I don't see any reason to not change it in the English translation as well, go for it!

File: web/public/locales/sv.json [hunk 5]
```diff
@@ -249,11 +253,11 @@
   "aboutUs": {
     "groupName": "Om oss",
     "whoAreYou-question": "Vilka är ni?",
-    "whoAreYou-answer": "Electricity Map är utvecklad och förvaltad av <a href=\"https://electricitymap.org/\" target=\"_blank\">Electricity Maps</a>, ett litet danskt nystartat företag. Vårt mål är att hjälpa mänskligheten att nå ett hållbart samhälle genom att samla in och tillgängliggöra klimatpåverkan från våra dagliga val.",
+    "whoAreYou-answer": "Appen är utvecklad och förvaltad av <a href=\"https://electricitymap.org/\" target=\"_blank\">Electricity Maps</a>, ett litet danskt nystartat företag. Vårt mål är att hjälpa mänskligheten att nå ett hållbart samhälle genom att samla in och tillgängliggöra klimatpåverkan från våra dagliga val.",
```
Comment (VIKTORVAV99) [inline web/public/locales/sv.json:256]:
```suggestion
    "whoAreYou-answer": "Appen är utvecklad och förvaltad av <a href=\"https://electricitymaps.com/\" target=\"_blank\">Electricity Maps</a>, ett litet danskt nystartat företag. Vårt mål är att hjälpa mänskligheten att nå ett hållbart samhälle genom att samla in och tillgängliggöra klimatpåverkan från våra dagliga val.",
```

**Review by VIKTORVAV99 — COMMENTED** [review]
Looks like I missed one before, sorry about that. If it's electricitymap.org in the en translation feel free to change it there too. After that this is ready to merge.

File: web/public/locales/sv.json (hunk 5 above)
Comment (magol) [reply web/public/locales/sv.json:256]: It is multiple `electricitymap.org` in both `sv.json` and `en.json`, but no `electricitymaps.com`. 
This maybe should be changed in its own pull request if it is applied to all translations.

Comment (VIKTORVAV99) [reply web/public/locales/sv.json:256]: sounds like a plan!

**Review by VIKTORVAV99 — APPROVED** [review]

**Pull request merged**